lib/parsec/fileparse.py

```diff
@@ -117,6 +121,11 @@ def _concatenate(lines):
     maxline = len(lines)
     while index < maxline:
         line = lines[index]
+        # Raise an error if line has a whitespace after the line break
+        if re.match(_BAD_CONTINUATION_TRAILING_WHITESPACE, line):
+            msg = ("Syntax error line {0}: Whitespace after the line break "
```

Comment:
Should be "Whitespace after the line *continuation* character ", no?

Reply:
Indeed. I've changed this. :+1: